servers/services/src/main/java/org/projectnessie/services/cel/CELUtil.java

```diff
@@ -171,9 +171,6 @@ public static Object forCel(Object model) {
     if (model instanceof Operation) {
       return new OperationForCelImpl((Operation) model);
     }
-    if (model instanceof KeyEntry) {
```

Comment:
In order to keep the current way of doing things, how about we introduce a new class holding the attributes we need?

Reply:
i am missing the advantages of the current way i.e. whats the benefit of having an Object -> Object function?

if i introduce a new class it will have identical fields as `KeyEntryForCelImpl`
`forCel(ContentKey, Type)` sounds OK so i'll add that